### PR TITLE
Issue 1510 - Multiple Agents

### PR DESCRIPTION
**Pull request by linggao**
Signed-off-by: Nadim Jabbour <Nadim.Jabbour@ibm.com>

### CODE REVIEW

File: test/Makefile [hunk 1]
```diff
@@ -263,7 +278,7 @@ endif
 agbot-prereq-build-anax: $(ANAX_SOURCE)/anax
 	@echo "Configuring agbot prerequisites"
 	if [[ "$(shell docker images -q $(DOCKER_ANAX_K8S_INAME) 2>/dev/null)" == "" ]]; then \
-		cd $(ANAX_SOURCE) && make anax-k8s-image; \
+		cd $(ANAX_SOURCE) && make anax-k8s-image && make anax-image; \
```
Comment (dabooz) [inline test/Makefile:281]: I ran with sall pattern and MUL_AGENTS=3, but the anax image container was not built, so the agents didnt run. This is the first problem. The second problem is that make test did not fail when the extra agents failed to start.

**Review by dabooz — CHANGES_REQUESTED** [review]

File: test/Makefile (hunk 1 above)
Comment (linggao) [reply test/Makefile]: fixed.

File: test/Makefile [hunk 2]
```diff
@@ -201,8 +210,14 @@ clean:
 	gov/stop_kube.sh
 	@echo "Shutdown anax cleanly"
 	-@curl -sSLX DELETE http://localhost:8510/node 2>/dev/null || true
+	-@curl -sSLX DELETE http://localhost:8512/node 2>/dev/null || true
```
[the remark this answers is not in the document]
Comment (linggao) [reply test/Makefile]: removed. they are leftover code.

**Review by dabooz — CHANGES_REQUESTED** [review]

File: test/gov/verify_surfaced_error.sh [hunk 3]
```diff
@@ -121,7 +121,7 @@ do
   NUM_ERRS=$(echo ${ERRS} | jq -r '. | length')
   sleep 5s
   ((TIMEOUT++))
-  if [[ $TIMEOUT == 26 ]]; then echo -e "surface error failed to resolve"; exit 2; fi
+  if [[ $TIMEOUT -ge 25 ]]; then echo -e "surface error failed to resolve"; exit 2; fi
```
[the remark this answers is not in the document]
Comment (linggao) [reply test/gov/verify_surfaced_error.sh]: Changed to 50.

**Review by dabooz — APPROVED** [review]

**Pull request merged**